### PR TITLE
:book: Added data license info for the API to the README.

### PR DESCRIPTION
**Pull request by torgo**
Related to #2713.

#### What kind of change does this PR introduce?

(Is it a bug fix, feature, docs update, something else?)

- [ ] PR title follows the guidelines defined in our [pull request documentation](https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process)

#### What is the current behavior?

No license listed for API data.

#### What is the new behavior (if this is a feature change)?**

No new behaviour - but explicitly document that the API data is provided under a cc0 license.

#### Special notes for your reviewer

This is a "straw man" proposal. It may be that we need additional data license artefacts as well.

#### Does this PR introduce a user-facing change?

For user-facing changes, please add a concise, human-readable release note to
the `release-note`

(In particular, describe what changes users might need to make in their
application as a result of this pull request.)

```release-note

```

### CODE REVIEW

**Comment by naveensrinivasan** [comment]
@torgo Thanks for the PR.

**Comment by david-a-wheeler** [comment]
It's probably fine. I think I'll need to check with LF legal. GitHub doesn't try to impose any license on data from its API, does it? Anyone know?

**Comment by david-a-wheeler** [comment]
Three points:

* Quick nit: the license short name is usually in all-caps, that is, it should say "CC0" not "cc0".
* I notice that this PR is to address issue https://github.com/ossf/scorecard/issues/2713.
* Please wait and do *not* accept this PR until we hear back from legal review. I've filed this as legal review LR-1558. I suspect it's fine but it's important to hear from actual legal experts first :-).

File: README.md [hunk 1]
```diff
@@ -150,6 +150,8 @@ To enable your project to be available on the REST API, set
 [`publish_results: true`](https://github.com/ossf/scorecard-action/blob/dd5015aaf9688596b0e6d11e7f24fff566aa366b/action.yaml#L35)
 in the Scorecard GitHub Action setting.
 
+Data provided by the REST API is [licenced cc0](https://creativecommons.org/share-your-work/public-domain/cc0/).
```
Comment (torgo) [inline README.md:153]:
```suggestion
Data provided by the REST API is [licenced CC0](https://creativecommons.org/share-your-work/public-domain/cc0/).
```

**Comment by torgo** [comment]
Thank you @david-a-wheeler!

> * Quick nit: the license short name is usually in all-caps, that is, it should say "CC0" not "cc0".

 I've added a suggested change to reflect this. Please feel free to accept.

I await

**Comment by torgo** [comment]
> > I await.
> 
> Me too :-). . I've filed this as legal review LR-1558.

Not sure what happened there. :/ But you got the gist.

**Comment by david-a-wheeler** [comment]
I expect to hear more sometime tomorrow. It may be them asking more questions, but let's try to get this resolved.

**Comment by david-a-wheeler** [comment]
See: https://github.com/ossf/scorecard/issues/2713

File: README.md (hunk 1 above)
Comment (torgo) [reply README.md:153]: Updated as discussed in #2713.

Comment (david-a-wheeler) [reply README.md:153]: +1

**Comment by spencerschrock** [comment]
@torgo Would you mind applying the CDLA suggestion, and then I'll get this merged in?

**Comment by david-a-wheeler** [comment]
FYI: this repo is configured so that I have to merge "main" into this branch before it can be (auto) merged into main. So I did that to get this to be merged.

Thanks for your persistence. Ensuring that everything is legal is very important.

**Comment by david-a-wheeler** [comment]
Grr. The auto-merge adds a merge commit (good!), but the merge commit doesn't have a DCO, so it's failing the DCO check even though there's no content to sign for DCO. Let me see if I can override this.

**Review by david-a-wheeler — APPROVED** [review]

**Comment by david-a-wheeler** [comment]
I don't see why this PR isn't passing the checks (other than DCO).

So I've re-created the PR, but included in the comment credit to @torgo . See: https://github.com/ossf/scorecard/pull/3404
Maybe with a re-created PR we can get this merged!

**Comment by david-a-wheeler** [comment]
There was some weird technical problem, so this PR was re-created here: https://github.com/ossf/scorecard/pull/3404 and then merged.

Thank you @torgo !

**Pull request closed without merging**